doc/docs.md

```diff
@@ -4764,6 +4764,21 @@ fn legacy_function() {}
 fn inlined_function() {
 }
 
+// This function's calls will NOT be inlined.
+[noinline]
+fn function() {
+}
+
+// This function will NOT return to its callers.
+// Such functions can be used at the end of or blocks,
+// just like exit/1 or panic/1. Such functions can not
```

Comment:
what does /1 mean?

Reply:
arity/number of arguments of the function